### PR TITLE
[cinder-csi-plugin][manila-csi-plugin] Make cloud-config/cacert volume/volumeMounts configurable

### PR DESCRIPTION
**Pull request by brtknr**
**What this PR does / why we need it**:

**Which issue this PR fixes(if applicable)**:
fixes #

**Special notes for reviewers**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: charts/cinder-csi-plugin/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: latest
 description: Cinder CSI Chart for OpenStack
 name: openstack-cinder-csi
-version: 1.2.0
+version: 1.2.2
```
Comment (jichenjc) [inline charts/cinder-csi-plugin/Chart.yaml:5]: just curious on the bump from .0 to .2 , any reason behind it?

Comment (ramineni) [reply charts/cinder-csi-plugin/Chart.yaml:5]: @jichenjc this PR , https://github.com/kubernetes/cloud-provider-openstack/pull/1211 , bumps to 1.2.1

**Comment by brtknr** [comment]
I am increasingly coming to the conclusion that the default appVersion on the master branch should be the last stable release but we can define instructions for users to override with latest tag. Backporting every little change to release-1.19 branch seems like unnecessary hassle.

**Comment by jichenjc** [comment]
> I am increasingly coming to the conclusion that the default appVersion on the master branch should be the last stable release but we can define instructions for users to override with latest tag. Backporting every little change to release-1.19 branch seems like unnecessary hassle.

+1 to this suggestion

**Comment by ramineni** [comment]
> I am increasingly coming to the conclusion that the default appVersion on the master branch should be the last stable release but we can define instructions for users to override with latest tag. Backporting every little change to release-1.19 branch seems like unnecessary hassle.

@brtknr I dont agree, because directly updating to latest tag might not be compatible with the 1.19 sidecars everytime as I mentioned before also. And also , generally we dont backport fixes to 1.19 unless it is required / needed . This time we missed the changes which actually need to be on 1.19, as last min changes, otherwise I dont think there will be much updates to release branches

**Comment by brtknr** [comment]
> > I am increasingly coming to the conclusion that the default appVersion on the master branch should be the last stable release but we can define instructions for users to override with latest tag. Backporting every little change to release-1.19 branch seems like unnecessary hassle.
> 
> @brtknr I dont agree, because directly updating to latest tag might not be compatible with the 1.19 sidecars everytime as I mentioned before also. And also , generally we dont backport fixes to 1.19 unless it is required / needed . This time we missed the changes which actually need to be on 1.19, as last min changes, otherwise I dont think there will be much updates to release branches

Hi @ramineni I am not clear on what you mean by 1.19 sidecars and why there would be compatibility issues there. Could you point me to the portion of the code you are speaking about? Thanks

File: charts/cinder-csi-plugin/templates/nodeplugin-daemonset.yaml [hunk 2]
```diff
@@ -74,12 +74,12 @@ spec:
...
               mountPropagation: "HostToContainer"
             - name: cacert
-              mountPath: /etc/cacert
+              mountPath: /etc/kubernetes
               readOnly: true
       volumes:
         - name: cacert
           hostPath:
-            path: /etc/cacert
+            path: /etc/kubernetes
```
Comment (ramineni) [inline charts/cinder-csi-plugin/templates/nodeplugin-daemonset.yaml:82]: @brtknr what is the reason for this change?

**Comment by ramineni** [comment]
@brtknr Still the CI job fails , is this expected? could you let us the reason for the failure

**Comment by ramineni** [comment]
> > > I am increasingly coming to the conclusion that the default appVersion on the master branch should be the last stable release but we can define instructions for users to override with latest tag. Backporting every little change to release-1.19 branch seems like unnecessary hassle.
> > 
> > 
> > @brtknr I dont agree, because directly updating to latest tag might not be compatible with the 1.19 sidecars everytime as I mentioned before also. And also , generally we dont backport fixes to 1.19 unless it is required / needed . This time we missed the changes which actually need to be on 1.19, as last min changes, otherwise I dont think there will be much updates to release branches
> 
> Hi @ramineni I am not clear on what you mean by 1.19 sidecars and why there would be compatibility issues there. Could you point me to the portion of the code you are speaking about? Thanks

https://github.com/kubernetes/cloud-provider-openstack/blob/master/manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml#L36-L62 , these are sidecars

**Comment by ramineni** [comment]
looks good. Thanks.
/lgtm

**Comment by jichenjc** [comment]
@ramineni is the PR https://github.com/kubernetes/cloud-provider-openstack/pull/1211 you talked about?
many PRs related to this fly around. so kind of lost context now

**Comment by ramineni** [comment]
> @ramineni is the PR #1211 you talked about?
> many PRs related to this fly around. so kind of lost context now
> 

ya, I understand, because of version , it has 1.2.1, so better to merge sequentially

**Comment by ramineni** [comment]
Thanks.
/lgtm
